robot/basestation/static/img/connection_icons/connected_bar.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=Link to Full Wi-Fi Signal
+Type=Link
+URL=https://cdn.pc.net/images/db/wi-fi_signal_full.png
+Icon=image-png
+Name[en_US]=connected_bar
```

Comment:
do we need the image in .desktop?

Reply:
removed connectFolder with images, not being used